app/workers/alert_mailer_worker.rb

```diff
@@ -3,14 +3,14 @@
 class AlertMailerWorker
   include Sidekiq::Worker
 
-  def self.schedule_email(alert:, user:)
-    perform_async(alert.id, user.id)
+  def self.schedule_email(alert)
+    perform_async(alert.id, alert.target_user.id)
```

Comment:
I'm following then pattern of the other workers and the `schedule_...` method receives AR objects, which negates the need to an object retrieval in that method.  (?)